code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -95,9 +95,33 @@
 /obj/effect/landmark/ert_spawns/distress_cryo
 	name = "Distress_Cryo"
 
-/datum/emergency_call/cryo_squad/tech
-	name = "Marine Cryo Reinforcements (Tech)"
-	mob_max = 5
-	max_engineers = 1
-	max_medics = 1
-	max_heavies = 0
+/proc/get_marines()
+    var/pop_scale = 0
+    var/count = 0
+
+    for (var/mob/living/carbon/human/H in GLOB.human_mob_list)
```

Comment:
```suggestion
    for (var/mob/living/carbon/human/H in GLOB.alive_human_list)
```

All humans completely or just alive ones?

Reply:
it says that it scales with POP not with alive marines tho

Reply:
already scales with humans either way /shrug

Reply:
ideally, it should scale off the whole server pop, but I'm not sure how to rewrite it for that